### ISSUE TITLE
Can't access Ombi on the local network if a base url is set up

### ISSUE THREAD

**Issue by LeVraiRoiDHyrule** [Ombi-app/Ombi#4046]
**Describe the bug**

If a base url is set up in Ombi settings (for Reverse proxy purposes), ombi no longer loads locally.

**To Reproduce**

Set a base url in the settings
Try to access [localip]:5000 or [localip]:5000/ombi
Ombi doesn't load

**Expected behavior**

Ombi should load on lan even with a base url, like Jellyfin for example.

**Screenshots**

![image](https://user-images.githubusercontent.com/64923542/107352017-a25f6c00-6acb-11eb-99e8-1a0b08978471.png)

**Logs (Logs directory where Ombi is located)**
Ombi didn't produce error logs in its install folder, how can I get more informations ?

**Desktop (please complete the following information):**
 - OS: DietPi on Raspberry Pi 4
 - 
**Ombi Version (please complete the following information):**
 - Version 4.0.1104
- Media Server Jellyfin
- Database Type: SQLite

**Comment by tidusjar** [comment]
I've been making some changes in this area recently, can you give this a go? https://dev.azure.com/tidusjar/Ombi/_build/results?buildId=1109&view=artifacts&pathAsName=false&type=publishedArtifacts

**Comment by jwhumphrey** [comment]
> I've been making some changes in this area recently, can you give this a go? https://dev.azure.com/tidusjar/Ombi/_build/results?buildId=1109&view=artifacts&pathAsName=false&type=publishedArtifacts

I'm not sure what was different in the win64 zip but this fixed my remote access issues in v4. Thank you!

**Comment by antetna** [comment]
I experienced similar issue with Linuxserver container, everything loading fine except one request that was not using basurl at all.

This one is stuck loading because waiting for credentials since it is outside of base path /ombi/ and that area is password protected by webserver. 
![Screenshot from 2021-02-27 19-23-33](https://user-images.githubusercontent.com/10363296/109396364-6e8c9f00-7931-11eb-9ed7-b2f42d7d44c1.png)

Tried that  build  and everything loads fine now. 

@tidusjar what are you plans on merging back this change back into development branch, anytime soon ?
